i18n/de/docusaurus-plugin-content-docs/current/help-center/your-account/account-settings.mdx

```diff
@@ -488,6 +488,31 @@ Abhängig davon in welchem Status sich das Chat-Backup befindet, haben Sie zwei
 
 In beiden Fällen, stellen Sie sicher, dass Sie die erstellte Passphrase sicher aufheben.
 
+## Chat-Einstellungen
+
+In diesem Abschnitt können Sie die Aktion „Drücken Sie die Eingabetaste“ ändern.
+
+<Tabs>
+  <TabItem value="web" label="Web" default>
+  <ol>
+    <li>Um die Einstellungen zu öffnen, klicken Sie auf Ihr Profilbild, oder Ihre Initialen.</li>
+    <li>Klicken Sie auf <b>Chat-Einstellungen</b>.</li>
+    <li>Standardmäßig wird durch Drücken der Eingabetaste eine neue Nachricht gesendet.</li>
+    <li>Wenn Sie eine neue Zeile durch Drücken der Eingabetaste beginnen möchten, klicken Sie auf die Option <b>Neue Zeile beginnen</b></li>
+    <li>Wenn Sie eine neue Nachricht durch Drücken der Eingabetaste senden möchten, klicken Sie auf die Option <b>Nachricht senden</b></li>
+  </ol>
+  </TabItem>
+  <TabItem value="mobile" label="Mobile">
+  <ol>
+    <li>Um die Einstellungen zu öffnen, klicken Sie auf Ihr Profilbild, oder Ihre Initialen.</li>
+    <li>Tippen Sie auf <b>Chat-Einstellungen</b>.</li>
+    <li>Standardmäßig wird durch Tippen auf die Eingabetaste eine neue Zeile begonnen.</li>
+    <li>Wenn Sie eine neue Zeile durch Drücken der Eingabetaste beginnen möchten, tippen Sie auf die Option <b>Neue Zeile beginnen</b></li>
+    <li>Wenn Sie eine neue Nachricht durch Drücken der Eingabetaste senden möchten, tippen Sie auf die Option <b>Nachricht senden</b></li>
```

Comment:
Add a dot at the end +
Change "neue Nachricht" to "Nachricht" since the message doesn't have to be new

Reply:
Fixed.